manifests/templates/registry-file-host.yaml.in

```diff
@@ -0,0 +1,121 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata: 
+        creationTimestamp: null
+        labels:
+                io.kompose.service: lb-claim0
```

Comment:
is it needed?

Reply:
Dictates what is the storage requirement for registry volume mount - 100M

Reply:
@zvikorn I mean the label.

Reply:
Sure - changing label to registry.vol.pvc